core/common/utils.py

```diff
@@ -23,3 +30,10 @@ def tldextract_parser(url):
         logging.error(e)
 
     return parts
+
+def parse_date_to_utc(date):
+    """
```

Comment:
What is `date`? a datetime object? a string?

This is a general remark (I've seen this in other places in your code) but it would be good to change the way you write docstrings.


```python
def parse_date_to_utc(date):
"""Parses a string to a datetime object in UTC timezone.

Args:
  date (str): string-timestamp to convert to datetime object. 
      e.g. <give an example of the expected format>

Returns:
  A datetime.datetime object in UTC timezone.
"""
    return parser.parse(...)
```